src/views/AddItem.jsx

```diff
@@ -1,3 +1,104 @@
+import React from 'react';
```

Comment:
A part of the suggestion for my next comment.

```suggestion
import React, { useState } from 'react';
```